libs/domain/picking/src/components/customer-note-modal/customer-note-modal.component.ts

```diff
@@ -0,0 +1,41 @@
+import { ButtonType } from '@spryker-oryx/ui/button';
+import { IconTypes } from '@spryker-oryx/ui/icon';
+import { CLOSE_EVENT } from '@spryker-oryx/ui/modal';
+import { i18n, Size } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { property } from 'lit/decorators.js';
+
+export class CustomerNoteModalComponent extends LitElement {
+  @property() note?: string;
```

Comment:
Why not making note a slot?
Also it will be easier to have `open` property in the same way that standard modal has it so it's similar to work with.